[flaky test] [reciver/iis] Flaky test TestIisIntegration

Seen in https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/3198868093/jobs/5224152682

```
=== CONT  TestIisIntegration
    integration_test.go:57: 
        	Error Trace:	D:\a\opentelemetry-collector-contrib\opentelemetry-collector-contrib\receiver\iisreceiver\integration_test.go:57
        	Error:      	Received unexpected error:
        	            	the following errors occurred:
        	            	 -  missing expected resource with attributes: map[iis.application_pool:---1]
        	            	 -  extra resource with attributes: map[iis.application_pool:---2]
        	Test:       	TestIisIntegration
--- FAIL: TestIisIntegration (1.66s)
```

--- a/receiver/iisreceiver/integration_test.go
+++ b/receiver/iisreceiver/integration_test.go
@@ -54,5 +54,7 @@ func TestIisIntegration(t *testing.T) {
 	expectedMetrics, err := golden.ReadMetrics(expectedFile)
 	require.NoError(t, err)
 
-	require.NoError(t, scrapertest.CompareMetrics(expectedMetrics, actualMetrics, scrapertest.IgnoreMetricValues()))
+	require.NoError(t, scrapertest.CompareMetrics(expectedMetrics, actualMetrics,
+		scrapertest.IgnoreResourceAttributeValue("iis.application_pool"),
+		scrapertest.IgnoreMetricValues()))
 }

